### DISCUSSION TITLE
Is there a way to hide the line-numbers while keeping the gutter padding?

### DISCUSSION THREAD

**Discussion by well1791** [helix-editor/helix#4145] [Q&A] [answered]
So far I see that the only options available for `line-number` are `relative` and `absolute` [here](https://github.com/helix-editor/helix/blob/6cca7375ec1965e61c791bbe738c617ea43c6dce/helix-view/src/editor.rs#L415), so, I wonder if there is any other way to hide them.

**Comment by the-mikedavis** [comment, marked as answer]
You can configure which elements are rendered in the gutter with the `editor.gutter` configuration option: https://docs.helix-editor.com/configuration.html#editor-section
